client/extensions/woocommerce/app/store-stats/index.js

```diff
@@ -99,20 +91,24 @@ class StoreStats extends Component {
 										.format( 'YYYY-MM-DD' )
 								: selectedDate
 						}
-						query={ query }
+						query={ orderQuery }
 						statsType="statsOrders"
 						showQueryDate
 					/>
 				</StatsPeriodNavigation>
 				{ config.isEnabled( 'woocommerce/extension-referrers' ) && (
 					<div>
 						{ siteId && (
-							<QuerySiteStats statType="statsStoreReferrers" siteId={ siteId } query={ query } />
+							<QuerySiteStats
+								statType="statsStoreReferrers"
+								siteId={ siteId }
+								query={ orderQuery }
```

Comment:
We should pull out `referrerQuery` above on line 52, and update the usage here and on lines 111 & 122 below. I know the query ultimately ends up being the same if no overrides are passed, but the naming will make more sense.

Reply:
Makes sense, will fix